### PR TITLE
[V5.0.1] EN-2188 

### PR DESCRIPTION
**Pull request by fedelocci**
show information from group references

### CODE REVIEW

File: src/ui/groups/detail/GroupDetailTabContents.js [hunk 1]
```diff
@@ -25,14 +26,16 @@ class GroupDetailTabContents extends React.Component {
...
   renderRows() {
-    return this.props.contentReferences.map(item => (
-      <tr key={item.code}>
-        <td>{item.name}</td>
-        <td>{item.code}</td>
-        <td>{item.type}</td>
-        <td>{item.lastEdit}</td>
-      </tr>
-    ));
+    const DATE_FORMAT = 'DD/MM/YYYY';
```
Comment (nicpuddu) [inline src/ui/groups/detail/GroupDetailTabContents.js]: lets not declare constants that are only used once.

**Comment by entando-service** [comment]
All is well

File: src/ui/groups/detail/GroupDetailTabContents.js (hunk 1 above)
Comment (fedelocci) [reply src/ui/groups/detail/GroupDetailTabContents.js]: i've fixed the component form your suggestion @nicpuddu

**Comment by entando-service** [comment]
All is well

**Pull request merged**